Sources/Build/describe().swift

```diff
@@ -128,6 +128,11 @@ public func describe(prefix: String, _ conf: Configuration, _ modules: [Module],
     #if os(Linux)
          args += ["-fPIC"]
     #endif
+    #if os(OSX)
+        if let sysroot = Toolchain.sysroot {
```

Comment:
I don't remember off the top of my head, but is sysroot only defined on OS X? If not, if it is just only frequently used on OS X, then there should be no need for the `#if`s here.